subprojects/docs/docs.gradle

```diff
@@ -402,7 +402,7 @@ releaseNotesMarkdown.configure {
     group = "release notes"
 }
 
-def releaseNotes = tasks.register("releaseNotes", Copy) {
+def releaseNotes = tasks.register("releaseNotes", Sync) {
```

Comment:
Doesn't that delete everything in the `docsDir`? I guess a simple copy should be enough here.